packages/telemetry/src/slog-to-otel.js

```diff
@@ -347,60 +406,68 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
     now = floatSecondsToHiRes(nowFloat);
     currentAttrs = slogAttrs;
 
-    if (initing) {
-      // Initialize the top-level span with the first timestamped slog entry.
-      spans.push('init');
-      initing = false;
-    }
-
     if (finished) {
       return;
     }
 
     // console.log('slogging', obj);
     switch (slogType) {
+      case 'kernel-init-start': {
+        spans.push('init');
+        break;
+      }
+      case 'kernel-init-finish': {
+        spans.pop('init');
+        break;
+      }
+      case 'bundle-kernel-start': {
+        assert.equal(spans.topKind(), 'init');
```

Comment:
All of these `assert`s.. this is in `slogSender()`, right? What happens if one of these throws, does the kernel panic? It looks like `writeSlogObject()` does not protect against `slogSender()` throwing, and `writeSlogObject` is called from all over.

Reply:
good, and that's on trunk now, so I won't worry about kernel-panic risk coming from any changes here